### DISCUSSION TITLE
Please don't include go patch versions in go module

### DISCUSSION THREAD

**Discussion by vijaynallagatla** [evilmartians/lefthook#725] [Ideas]
We recently encountered breaking change in version of the go module in our project due to including the latest `lefthook` version `v1.6.11`.

This forces user's to bump their go version to `v1.22.2`. It'd be good if you can just include `go 1.22` unless you are dependent on patch release.

**Comment by mrexox** [comment]
I've removed the patch version from `go.mod`. Going to release it with the next lefthook version.
